### ISSUE TITLE
Bug: mailspring gets blank after configure IMAP/SMTP account

### ISSUE THREAD

**Issue by Alexander-Huang** [Foundry376/Mailspring#1878]
**Describe the bug**
After configure IMAP/SMTP account,The mailspring becomes blank.Waiting for minutes,nothing appears.

And this is the command output when I run it in a terminal:
```
Running database migrations
App load time: 340ms

{"error":null}
Manual update check (updates.getmailspring.com/check/linux/x64/1.7.4-d54ff730/d71ded53-1542-45fb-ba91-f800238a5def/stable) returned 204
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
Raven: undefined - undefined
Raven: undefined - undefined
```

**Expected behavior**
It shows options of interface configuration and hotkey settings after that.

**Screenshots**
This is how I configure it:
![2020-03-29 21-01-51 的屏幕截图](https://user-images.githubusercontent.com/42209822/77849764-98f54a00-7200-11ea-94c9-c0eae8009298.png)
And the blank interface:
![2020-03-29 21-03-25 的屏幕截图](https://user-images.githubusercontent.com/42209822/77849791-c0e4ad80-7200-11ea-9cbb-9bf4d794d874.png)

**Setup: (please complete the following information):**
 - OS: Arch Linux
 - OS Version 2020-03-29(Sorry,Archlinux doesn't have versions)
 - Mailspring Version: 1.7.4-1

**Comment by xmoroccan** [comment]
I have the same problem, with Microsoft live mail service, on fedora 31
![Screenshot from 2020-03-29 21-30-47](https://user-images.githubusercontent.com/20878658/77860017-a57ba180-7204-11ea-8b0c-bf28d304e899.png)

**Comment by gachowy** [comment]
Same here, first email account, SMTP/IMAP:
```
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
```

OS: Fresh Ubuntu 19.10 (5.3.0-42-generic #34-Ubuntu SMP Fri Feb 28 05:49:40 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux)
Mailspring: 1.7.4-d54ff730

**Comment by dzakharov90** [comment]
I have same problem on Ubuntu\Xubuntu 19.10

**Comment by siddhantmahalle** [comment]
I have the same problem on Ubuntu 18.04

**Comment by 20Web** [comment]
Until a fix is provided see:
https://github.com/Foundry376/Mailspring/issues/1847#issuecomment-596273025

**Comment by alfredo87** [comment]
This is what happens to me: on manjaro the app just shut down with a cryptic message about json. Then all accounts were reset and re connecting them did not work either. I am not giving any more details because a client that does crash like that is useless!! I am uninstalling it ....

**Comment by redac** [comment]
Same problem, Windows 10 Pro.

**Comment by skrive** [comment]
This bug was fixed in #1876, but there is no release with this update yet.

You can install 1.7.2 as a temporary workaround.

**Comment by Alexander-Huang** [comment]
Okay.Considering that there is a certain way to fix it and I don't want to receive mails anymore,I'll close this issue.

**Comment by gabyferman** [comment]
I'm having the same problem. I'm using Windows 10 with Mailspring version 1.7.4-d54ff730

Any news about a fix for this issue?

**Comment by skrive** [comment]
> I'm having the same problem. I'm using Windows 10 with Mailspring version 1.7.4-d54ff730
> 
> Any news about a fix for this issue?

It's already been fixed but hasn't been incorporated into a release yet.

The best thing to do at the minute is to install 1.7.2, add all your IMAP accounts then install 1.7.4.

**Comment by bengotow** [comment]
Hey folks! Thanks for filing this issue—sorry it's taken so long for the fix in PR #1876 to get shipped. I will cut a release tomorrow that includes the fix - I appreciate all the help tracking down the problem!

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀 The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
